app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -250,8 +264,8 @@
             android:textColor="@color/cast_text"
```

Comment:
what about changing marginTop for texts?
maybe it's better to constant all texts
top_toTopOf"@+id/firstTextId"?